WIP-SQL/types-of-database-servers.md

```diff
@@ -0,0 +1,69 @@
+# Database server types
+author: SebaRaba
+
+levels:
+
+  - beginner
+
+  - basic
+
+type: normal
+
+category: must-know
+
+links:
+
+  - '[link to learn more](https://enki.com)'
+
+---
+## Content
+
+Depending on the user's needs, we can distinguish different types of database servers.
```

Comment:
Change to "There are a number of RDBMS servers available, each with different strengths"

Reply:
Read this article: https://en.wikipedia.org/wiki/Comparison_of_relational_database_management_systems
Then, update the list of features under each DB type. Focus on fundamental features, which is everything from here:
https://en.wikipedia.org/wiki/Comparison_of_relational_database_management_systems#Fundamental_features to the end.